scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,148 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
+# https://github.com/Azure/pcs-cli/blob/master/remotemonitoring/single-vm/docker-compose.dotnet.yml
+#
+# Usage:
+# - install Docker Compose, see https://docs.docker.com/compose/install
+# - open a console and `cd` into the folder that contains this file
+# - note how this file references many environment variables, either set
+#   them in the .env file or edit this file adding the values here
+# - run `docker-compose up`
+# - open the browser at http://127.0.0.1:10443
+#
+# For more information see
+# https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet
+
+version: "3"
+
+services:
+  reverseproxy:
+    image: azureiotpcs/remote-monitoring-nginx:testing
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on:
+      - webui
+      - auth
+      - iothubmanager
+      - devicesimulation
+      - telemetry
+      - config
+    volumes:
+    # Required volume for app certs
+      - ./app/certs:/app/certs:ro
+
+  webui:
+    image: azureiotpcs/pcs-remote-monitoring-webui:testing
+    depends_on:
+      - auth
+      - iothubmanager
+      - devicesimulation
+      - telemetry
+      - config
+  # NOTE: Volumes disabled for local development, uncomment if enabling auth
+  # volumes:
+  #   - ./webui-config.js:/app/build/webui-config.js:ro
+
+  auth:
+    image: azureiotpcs/pcs-auth-dotnet:testing
+    environment:
+      - PCS_AUTH_ISSUER
```

Comment:
for each of these variables should we include examples similar to what you're doing above?

Reply:
I think having it in the .env file is good enough since our tutorial asks you to look for that and modify that file, not the docker-compose one.